controllers/policy_controller.go

```diff
@@ -125,35 +132,46 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	// RECONCILE THE RESOURCE
 
-	// if there is already an ARN in our status, then we update the object
+	// We try to create the resource
 	statusWriter, err := CreateAWSObject(iamsvc, ins, DoNothingPreFunc)
 	statusWriter(ctx, ins, &policy, r.Status(), log)
 	if err != nil {
```

Comment:
We can simplify this, by turning the condition around. If err == nil, then we log creation success and return. Then we don't have to have the whole error handling inside another if. Can be cleaner?

Reply:
this refers to the error handler from 131 onwards... not sure why my comment here ended up being weird